src/pages/en/core-concepts/astro-components-new.md

```diff
@@ -102,8 +115,11 @@ CSS `<style>` tags are also supported inside of the component template. They can
   /* scoped to the component, other H1s on the page remain the same */
   h1 { color: red }
 </style>
+
 <h1>Hello, world!</h1>
-```
+``` 
+
+> ⚠️ The styles defined here apply only to content written directly in the component's own component template. Children, and any imported components will **not** be styled by default. See our [Styling Guide](/en/guides/styling) for more information on applying styles.
```

Comment:
worth moving "See our [Styling Guide](/en/guides/styling) for more information on applying styles" out of this quote and into the top level to match the other "📚 ..." paragraphs on the page?

Reply:
Done.